07week/dom-practice/js/script.js

```diff
@@ -1,5 +1,30 @@
-'use strict';
-
-document.addEventListener("DOMContentLoaded", function(event) {
-  // You code here
-});
+'use strict';
+document.addEventListener('DOMContentLoaded', () => {
+  // Your code here
```

Comment:
I also do not see where the part that counts how many items are available and displays that.